src/events/2023jodoseminar.md

```diff
@@ -0,0 +1,256 @@
+  <iframe class="gmap" src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2689.2350222196287!2d-122.3206264!3d47.62156259999999!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x5490153b7de58cf1%3A0x83445694c63803ef!2sOlmstead!5e0!3m2!1sen!2sus!4v1680641487123!5m2!1sen!2sus" width="400" height="300" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
+</section>
+
+<section>
+  <h2>Parking</h2>
+
+  <p>Parking is available through Seattle University. You can register now for a discounted price, or pay a higher fee if you
+    decide when you arrive. The standard rate for parking is $18 per calendar day if you don't pre-pay. Either way, we do not need
+    your car's registration information until you arrive.</p>
+
+  <p>Street parking in the area is very limited. We strongly recommend parking with Seattle University if you are driving.</p>
+
+  <p>We'll send more information closer to the date about parking with Seattle University - pre-paid or not.</p>
+
+</section>
+
+<section>
+  <h2>Dietary and Special Accommodations</h2>
+
+  <p>Please let us know in the "Dietary and Special Accommodations" section of the registration form if you require any
+    special accommodation fully participate in the seminar. Hoshu Dojo is responsible for notifying the University of any requests
+    by seminar participants with disabilities for reasonable accommodation that would require modification of University
+    facilities. Such notification must be made sufficiently in advance of the seminar for the University to consider the request,
+    determine whether the requested modification constitutes a reasonable accommodation or creates an undue hardship, and enter a
+    dialogue regarding the requested accommodation, if necessary. The University is not responsible for making the content of the
+    seminar accessible to persons with disabilities, such as providing interpreters for the deaf or assistance to the
+    vision-impaired.</p>
+
+  <p>Please also note that Seattle University dining hall kitchens and the venue for Saturday night's dinner are not kosher or
+    gluten free. Seattle University does have a station that avoids gluten but the kitchen still works with gluten in other areas.
+    This may be okay for people avoiding gluten for dietary or philosophical reasons, but might cause problem for people with
+    Celiac disease.</p>
+
+  <p>Please let us know in the "Dietary and Special Accommodations" section of the registration form if any of this affects you so
+    we can discuss arrangements.</p>
+
+</section>
+
+<section>
+  <h2>Seattle University is a non-smoking campus</h2>
+
+  <p>Smoking, vaping, and tobacco in any form are strictly prohibited everywhere on campus. Please find a public street off-campus
+    to smoke, vape, chew tobacco, etc.</p>
+</section>
```

Comment:
maybe including another copy of the "Register now" button down here would be good, so lazy people don't need to scroll back up.

Reply:
Good idea - Done.